README.md

```diff
@@ -3,7 +3,60 @@
 The `metabase-materialize-driver` allows [Metabase](https://github.com/metabase/metabase) 
 to connect to an instance of Materialize.
 
-## How to use the driver
+![Choose Materialize from database dropdown](images/materialize-metabase.png)
+
+## To Use the Driver
+
+To use the `metabase-materialize-driver`, put a copy of the driver's
+.jar file into the `/plugins` directory of your Metabase instance. Metabase
+will register the driver automatically! (For deployment-specific details, 
+please consult the following sections.)
+
+Once the Materialize driver is registered, use the following information
+to connect: 
+
+| Field             | Value     |
+| ----------------- |:---------:|
+| Host              | localhost |
+| Port              | 6875      |
+| Database username | default   |
+| Database password | default   |
```

Comment:
are these all the defaults? If so, should we maybe omit lines 15-23, and just tell people that the `materialize` database type should be available?

Reply:
these are the defaults! happy to remove

Reply:
I am nervous people won't trust the defaults.